src/views/preview/contact-preview-content.jsx

```diff
@@ -265,7 +265,13 @@ function ContactPreviewContent({ contact, onEdit, onDelete, onPrint, onArchieve,
 						mainAlignment="flex-end"
 						padding={{ horizontal: 'extrasmall' }}
 					>
-						{contact.parent !== '3' && <IconButton icon="MailModOutline" onClick={onMail} />}
+						{contact.parent !== '3' && (
+							<IconButton
+								icon="MailModOutline"
+								onClick={onMail}
+								disabled={Object.keys(contact?.email).length < 1}
```

Comment:
here using === 0 will be better ,also I have a question  contact.email is object ?

Reply:
To check if it is an empty object we can use the isEmpty function from lodash (seems safer than this approach)

Reply:
> here using === 0 will be better ,also I have a question contact.email is object ?

OK updated it and used isEmpty function. Yes contact.email is object.